htdocs/install/mysql/migration/9.0.0-10.0.0.sql

```diff
@@ -27,5 +27,5 @@
 -- To set a field as default NULL:             -- VPGSQL8.2 ALTER TABLE llx_table ALTER COLUMN name SET DEFAULT NULL;
 -- Note: fields with type BLOB/TEXT can't have default value.
 
-
-ALTER TABLE llx_facture CHANGE facnumber ref VARCHAR(30) NOT NULL;
\ No newline at end of file
+ALTER TABLE llx_facture CHANGE facnumber ref VARCHAR(30) NOT NULL;
+ALTER TABLE llx_facture DROP INDEX idx_facture_uk_facnumber, ADD UNIQUE idx_facture_uk_ref (ref, entity);
```

Comment:
@atm-maxime 
delete before change field name recreate after

ALTER TABLE llx_facture DROP INDEX idx_facture_uk_facnumber;
ALTER TABLE llx_facture CHANGE facnumber ref VARCHAR(30) NOT NULL;
ALTER TABLE llx_facture ADD UNIQUE idx_facture_uk_ref (ref, entity);

Reply:
Also, we must use 1 line for each action (no DROP..., ADD ... allowed by migration process)